backend/routers/energy.py

```
@@ -0,0 +1,105 @@
+    if not meter:
+        raise HTTPException(status_code=404, detail="Meter not found")
+    
+    db.delete(meter)
+    db.commit()
+    return {"message": "Meter deleted successfully"}
+
+# Consumption
+@router.post("/consumption")
+async def create_consumption(
+    consumption: schemas.ConsumptionCreate,
+    db: Session = Depends(get_db),
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)